Automattic/wp-calypso
# Onboarding WordPress importer: content everything flow

#### Changes proposed in this Pull Request

* Improved **content-chooser component** (disable Continue button until the info from the server is received)
* Implemented **Import Everything** flow (extended the class from the migration module with overwritten render UI logic)
* Implemented **Confirm screen** with the accompanying modal
* Moved common styles to the parent import styles.css file

#### Testing instructions

* Go to `/start/importer/capture?siteSlug={SLUG_ATOMIC}`, target website should be under the Business plan
* Enter the source website URL, example: `amused-swan.jurassic.ninja` (it has jetpack installed)
* From the ready screen, press the "Import your content" button
* Then select the "Everything" option by pressing the "Continue" button
* And finally, press the "Start import" button and confirm the modal
* Check if there is a proper Progress screen with a Hooray screen at the end

#### Screenshots
![Markup on 2022-01-19 at 13:22:55](https://user-images.githubusercontent.com/1241413/150129083-42e03863-1fa1-4d98-a739-4c8ed207fcbc.png)
![Markup on 2022-01-19 at 13:23:50](https://user-images.githubusercontent.com/1241413/150129087-c3969e25-9379-4e5c-ae11-cd4958bf9bd1.png)
![Markup on 2022-01-19 at 13:24:23](https://user-images.githubusercontent.com/1241413/150129092-8702eccd-c4cc-457b-97af-272a1b542a24.png)
![Markup on 2022-01-19 at 13:24:55](https://user-images.githubusercontent.com/1241413/150129094-6e15a828-078c-496d-9570-a0d086077bd5.png)
![Markup on 2022-01-19 at 13:25:30](https://user-images.githubusercontent.com/1241413/150129096-2d8dfce0-b758-403c-807c-c6ecaa6f8de5.png)

#### Known issues:
* #60237
* #60238
* #60241

Related to #59042
